### PR TITLE
add pid.available for evict-head on linux

### PR DESCRIPTION
**Pull request by Cweiping**
Fix Issue: https://github.com/kubernetes/kubernetes/issues/98206
Reference PR: https://github.com/kubernetes/kubernetes/pull/98207

/langage en

### CODE REVIEW

**Comment by bradtopol** [comment]
Hi @Huang-Wei can you do a technical review of this PR?  Simply add a lgtm if you feel it is technically correct. Thanks!
/cc @Huang-Wei 
/assign

**Comment by Huang-Wei** [comment]
Given that the dependent code PR https://github.com/kubernetes/kubernetes/pull/98207 will be delivered to k/k master (1.21), I guess this doc PR is more supposed to be delivered to `dev-1.21` branch instead of `master`? (as IIUC `master` in doc represents the latest stable release, which is 1.20)

@bradtopol can comment on the doc branch strategy.

File: content/en/docs/tasks/administer-cluster/out-of-resource.md [hunk 1]
```diff
@@ -116,9 +116,13 @@ The `kubelet` has the following default hard eviction threshold:
 
 * `memory.available<100Mi`
 * `nodefs.available<10%`
-* `nodefs.inodesFree<5%`
 * `imagefs.available<15%`
 
+Especially, on Linux also includes:
```
Comment (ehashman) [inline content/en/docs/tasks/administer-cluster/out-of-resource.md:121]:
```suggestion
On Linux, the following are also set:
```

**Comment by Cweiping** [comment]
@ehashman
It will be fixed in release v1.21.
https://github.com/kubernetes/website/pull/26261

**Comment by SergeyKanzhelev** [comment]
/reopen

Was it closed by mistake?

**Comment by Cweiping** [comment]
> /reopen
> 
> Was it closed by mistake?

It should be merged into the dev-1.21 branch. https://github.com/kubernetes/website/pull/26261

**Comment by SergeyKanzhelev** [comment]
> > /reopen
> > Was it closed by mistake?
> 
> It should be merged into the dev-1.21 branch. #26261

oh, sorry, it is an evening for me.

**Pull request closed without merging**